hugo/content/docs/tracing.md

```diff
@@ -10,148 +10,128 @@ title: Distributed Tracing
-Proto.OpenTracing has its own tags that you can use for Proto.Actor. Let's take a look at these tags.
-
-- `MessageType` - the name of the message type.
-- `TargetPID` - process ID of the message target.
-- `SenderPID` - process ID of the message sender.
-- `ActorPID` - current actor process ID. It equals TargetPID when this is a receiving Span, or SenderId when this is a sending Span.
-- `ActorType` - a type of the current actor.
-
-**Logs** are key-value pairs that describe a particular moment within the context of a Span. They are used to document a specific point in time. They can also be applied to the entire Span.
-
-**Baggage** items are key-value pairs that store additional useful information. This information can be transferred between different processes. Baggage items apply to a specified Span, Span Context, and all Spans that directly or transitively refer to a local Span.
-
-### SpanContext
-
-SpanContext can transfer data between processes. It consists of two parts: an implementation-dependent state to refer to the distinct Span within a trace
-and baggage items.
-
-### Scope
-
-**The scope** allows you to manage the state of Spans. It can activate and deactivate them since at the current time only one Span can be active.
-
-## Benefits of using OpenTracing
-
-- OpenTracing can be used with different distributed tracing systems, such as Zipkin, Jaeger, LightStep, DataDog, and many others.
-- It provides a standardized way to collect and map data from asynchronous interactions.
-- OpenTracing allows you to constantly collect up-to-date data from the runtime.
-- OpenTracing allows you to transfer data between different threads and processes.
-
-### Examples of using distributed tracing:
+### Examples of using distributed tracing
 
 - Monitoring the effectiveness of the application. When an application uses many services, using tracing, you can easily find out how long each service takes and where an exception is thrown.
 - Tracing allows you to know how fast requests are transmitted between different applications.
 - Tracking the history of the work of one process, which is simultaneously accessed by several services.
 - Tracking the history of request execution from start to finish.
 
-We have covered the benefits of using an OpenTracing and its basic concept and terminology. Now let's see how to use Proto.OpenTracing for Proto.Actor.
+We have covered the benefits of using an OpenTelemetry and its basic concept and terminology. Now let's see how to use Proto.OpenTelemetry for Proto.Actor.
 
-## Getting started with Proto.OpenTracing
+## Getting started with Proto.OpenTelemetry
 
-In order to start working with Proto.OpenTracing you need to install the Proto.OpenTracing package. To do this, you must install Visual Studio at least the 2019 version and .Net Core 3.0.
+[Realtime map is using Proto.OpenTelemetry](https://github.com/asynkron/realtimemap-dotnet/blob/main/Backend/Program.cs#L19) and might be used as a working example.
```

Comment:
Could be bad idea to link to line number on main branch. Link to specific commit instead?